marconi-chain-index/src/Marconi/ChainIndex/Experimental/Run.hs

```diff
@@ -67,7 +69,10 @@ run appName = withGracefulTermination_ $ do
       preferredStartingPoint = Cli.optionsChainPoint $ Cli.commonOptions o
 
   nodeConfigPath <- case Cli.optionsNodeConfigPath o of
-    Just cfg -> pure cfg
+    Just cfg -> do
+      exists <- doesFileExist cfg
+      unless exists $ error ("Config file does not exist at the provided path: " <> cfg)
```

Comment:
Use `logError` instead for printing the error message (for consistent logging), followed by `exitFailure`.